pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -110,7 +128,7 @@ let
         # (swap/viminfo) and redirect errors to stderr.
         # Only display the log on error since it will contain a few normally
         # irrelevant messages.
-        if ! $out/bin/nvim \
+        if ! "${neovim}/bin/nvim" \
```

Comment:
This changes it to call the unwrapped neovim, whereas previously it was wrapped with access to node/python/ruby. Does it matter for this not to be the case anymore for rplugin.vim generation?

Reply:
> The double-wrapping can't be removed this easily, because the previous code actually calls nvim to generate rplugin.vim after the first but before the second wrapping.

Indeed it isn't tricky at all. I didn't fully understand at first the functionality the wrapper should offer. I think I've finally managed to do it properly now. Currently `rplugin.vim` is generated at the end, as in the original design, and I've split the "calculation" of the flags to clean-nvim-flags vs `makeWrapperArgs` so that I _do_ call `${neovim}/bin/nvim` but with the flags and the environment the same as used in the wrapper (besides `NVIM_SYSTEM_RPLUGIN_MANIFEST` of course which _shouldn't_ be used along with `NVIM_RPLUGIN_MANIFEST`).